week_5/multi-step-form/src/app/components/heading/heading.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-heading',
+  standalone: true,
+  imports: [],
+  templateUrl: './heading.component.html',
+  styleUrl: './heading.component.css',
+})
+export class HeadingComponent {
+  @Input() headingTxt!: string;
```

Comment:
Txt compared to Text is just missing e. You don't use shortform of words where the only difference in using the full word is just one character

Reply:
Done